src/metabase/api/collection.clj

```diff
@@ -349,26 +349,53 @@
 (defn- collection-children*
   [collection models {:keys [sort-info] :as options}]
   (let [sql-order   (case sort-info
-                      nil                  [[:%lower.name :asc]]
-                      [:name :asc]         [[:%lower.name :asc]]
-                      [:name :desc]        [[:%lower.name :desc]]
-                      [:last-edited :asc]  [(if (= @mdb.env/db-type :mysql)
-                                              [(hsql/call :ISNULL :last_edit_timestamp)]
-                                              [:last_edit_timestamp :nulls-last])
-                                            [:last_edit_timestamp :asc]
-                                            [:%lower.name :asc]]
-                      [:last-edited :desc] (into
-                                            (case @mdb.env/db-type
-                                              :mysql
-                                              [[(hsql/call :ISNULL :last_edit_timestamp)]]
-                                              :postgres
-                                              [[(hsql/raw "last_edit_timestamp DESC NULLS LAST")]]
-                                              :h2
-                                              [])
-                                            [[:last_edit_timestamp :desc]
-                                             [:%lower.name :asc]])
-                      [:model :asc]        [[:model_ranking :asc]  [:%lower.name :asc]]
-                      [:model :desc]       [[:model_ranking :desc] [:%lower.name :asc]])
+                      nil                     [[:%lower.name :asc]]
+                      [:name :asc]            [[:%lower.name :asc]]
+                      [:name :desc]           [[:%lower.name :desc]]
+                      [:last-edited-at :asc]  [(if (= @mdb.env/db-type :mysql)
+                                                 [(hsql/call :ISNULL :last_edit_timestamp)]
+                                                 [:last_edit_timestamp :nulls-last])
+                                               [:last_edit_timestamp :asc]
+                                               [:%lower.name :asc]]
+                      [:last-edited-at :desc] (keep identity
+                                                    [(case @mdb.env/db-type
+                                                       :mysql
+                                                       [(hsql/call :ISNULL :last_edit_timestamp)]
+                                                       :postgres
+                                                       [(hsql/raw "last_edit_timestamp DESC NULLS LAST")]
```

Comment:
why are these raw statements instead of little happy fun packets of honeysql, anyhow?

Reply:
for the mysql branch, that is honeysql, calling the `ISNULL` function. For the postgres, honeysql1 cannot express `[:last_edit_timestamp :desc :nulls :last]` in the order-by clause but honeysql2 can. These two libraries can actually coexist on the classpath, but since we are using Toucan we can't easily get the sql generated by honeysql2. Thus the raw bit.

Reply:
`user=> (sql/format {:order-by [[:last_edit_timestamp :desc :nulls-last]]})
["ORDER BY last_edit_timestamp DESC NULLS LAST"]` works fine on my machine? is this a postgres-specific tihng?

Reply:
not sure. let me re-investigate. i had been trying (i think) `(hsql/format {:order-by [[:last_edit_timestamp :desc :nulls last]]})`

Reply:
https://clojurians-log.clojureverse.org/sql/2021-05-28 is when i was chatting about it

Reply:
looks like maybe the exact symbol just slipped his mind

Reply:
@howonlee we're gonna merge and maybe do a cleanup pass on this during this week